sycl/test-e2e/Matrix/SPVCooperativeMatrix/SG32/element_wise_all_ops_half.cpp

```diff
@@ -0,0 +1,23 @@
+//==----------- element_wise_all_ops_half.cpp  - DPC++ joint_matrix---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
+// UNSUPPORTED: gpu-intel-dg2
+// REQUIRES: aspect-fp16
+// REQUIRES: aspect-ext_intel_matrix, gpu
```

Comment:
I think this was added by mistake in original test.
```suggestion
// REQUIRES: aspect-ext_intel_matrix
```

Reply:
If I run the original copy of this test on CPU - it segfaults. Let me stay aligned with the original test and put REQUIRES: gpu

Reply:
Since you add `XFAIL: cpu`, it would be reported as expected fail, so I don't see why `REQUIRES: gpu` can't be removed.
Though, since original test has this mistake, I will not insist on fixing it here. It is ok to stay aligned.